clients/python/dd_client/__init__.py

```diff
@@ -255,6 +258,7 @@ def post_predict(
         parameters_output,
         use_base64=False,
         index_uris=[],
+        from_img=False
```

Comment:
you could remove "from_img" parameter and just detect whether the image contained in `data` is a `np.array` or a string

Reply:
Could be. One drawback is that it requires importing numpy... Already I don´t like having to import cv2 into the client.

Reply:
FTR, it's testing for `np.ndarray`.

Reply:
cv2 imports numpy anyways...

Reply:
cv2 image is a numpy array